app/forms/provider_interface/offer_wizard.rb

```diff
@@ -270,5 +317,79 @@ def create_method(name, &)
     def first_page_with_ske_feature_flag_disabled?(index)
       index.zero? && FeatureFlag.inactive?(:provider_ske) && decision.to_sym == :make_offer
     end
+
+    def course_subject_for_language_flow?
+      subject = course_option.course.subjects.first
+      MinisterialReport::SUBJECT_CODE_MAPPINGS[subject&.code] == :modern_foreign_languages
+    end
+
+    def ske_languages_selected
+      return if number_of_selected_ske_languages <= MAX_SKE_LANGUAGES
+
+      errors.add(:ske_language_required, :too_many, count: MAX_SKE_LANGUAGES)
+    end
+
+    def no_languages_selected
+      if number_of_selected_ske_languages > 1 && ske_language_required.include?('no')
+        errors.add(:ske_language_required, :no_and_languages_selected)
+      end
+    end
+
+    def number_of_selected_ske_languages
+      Array(ske_language_required).compact_blank.count
+    end
+
+    def ske_length_less_than_36_weeks
+      return if ske_language_length_1.to_i + ske_language_length_2.to_i <= 36
```

Comment:
can we adjust this such that, if the max number of SKE weeks changed, we would only have to change the value of a single constant? that means extracting `36` to `MAX_SKE_WEEKS` or some such, interpolating this in the error messages, and changing the name of this method.

Reply:
🆗